sickbeard/tvcache.py

```diff
@@ -179,14 +179,19 @@ def updateCache(self):
             return
 
         try:
+
+            # set updated before calling getRSS (it takes time to process all data)
+            # We should call setLastUpdate when the daily process start
+            # becase it checks if 'self.shouldUpdate' in the begining of the process
+            # and the getRSS run/process time (some seconds) will make it skip the next update
+            # as I showed in logs
+            self.setLastUpdate()
+
             data = self._getRSSData()
             if self._checkAuth(data):
                 # clear cache
                 self._clearCache()
 
-                # set updated
-                self.setLastUpdate()
```

Comment:
Don't know @fernandog. I think theres a reason you mark a search as 'updated' after you know for sure you can authenticate. You maybe could move it up the _clearCache, but that's about it.